fedimint-core/src/api.rs

```diff
@@ -968,10 +968,7 @@ impl<C: JsonRpcClient> WsFederationApi<C> {}
 /// The status of a server, including how it views its peers
 #[derive(Debug, Clone, Default, Serialize, Deserialize, PartialEq, Eq)]
 pub struct ConsensusStatus {
-    /// The last contribution that this server has made to the consensus, it's
-    /// equivalent to [`PeerConsensusStatus::last_contribution`] and
-    /// [`ConsensusContribution::value`]
-    pub last_contribution: u64,
+    pub epoch_count: u64,
```

Comment:
`contribution` was used as a generic term to keep the api backwards-compatible even if we removed "epochs". Will Aleph call it `epoch_count`?

Reply:
I will rename it to block_count then.